src/components/WeekDayPicker/__test__/weekDayPicker.spec.js

```diff
@@ -112,11 +116,24 @@ describe('<WeekDayPicker />', () => {
                 .prop('checked'),
         ).toBe(false);
     });
-    it('should call set useUniqueIdentifier for the days and the days wrapper element', () => {
-        useUniqueIdentifier.mockReset();
+    it('should call useUniqueIdentifier 8 times for the ids generations', () => {
+        useUniqueIdentifier.mockClear();
         mount(<WeekDayPicker />);
         expect(useUniqueIdentifier).toHaveBeenCalledTimes(8);
     });
+    it('should have default name when name params is not sent', () => {
+        const component = mount(<WeekDayPicker />);
+        expect(
+            component
+                .find(WeekDayItems)
+                .at(0)
+                .prop('name'),
+        ).toBe('week-day-items');
+    });
+    it('should have locale configured properly if locale is sent', () => {
+        const component = mount(<WeekDayPicker locale="en-US" />);
+        expect(component.prop('locale')).toBe('en-US');
+    });
```

Comment:
what is being tested here?
if you pass ```locale="en-US"``` to WeekDayPicker then you don't need to check that itself receive the locale prop